cms/cache/page.py

```diff
@@ -77,16 +78,28 @@ def set_page_cache(response):
             # We also store the absolute expiration timestamp to avoid
             # recomputing it on cache-reads.
             expires_datetime = timestamp + timedelta(seconds=ttl)
-            cache.set(
-                _page_cache_key(request),
-                (
-                    response.content,
-                    response._headers,
-                    expires_datetime,
-                ),
-                ttl,
-                version=version
-            )
+            if not DJANGO_3_2:
```

Comment:
As above, there's a lot of duplicated code for the header change. We should just use a new variable for the header I think. Something like, this also makes it much clearer for anyone cleaning out support logic in the future:

```
# support < django 3.2
if not DJANGO_3_2:
    response_headers = response._headers
# support >= django 3.2
else:
    response_headers = response.headers

cache.set(
    _page_cache_key(request),
    (
        response.content,
        response_headers,
        expires_datetime,
    ),
    ttl,
    version=version
)
```

Reply:
Righto!